frontend/package.json

```diff
@@ -21,15 +22,23 @@
     "@radix-ui/react-slot": "^1.0.2",
     "@radix-ui/react-switch": "^1.0.3",
     "@radix-ui/react-toggle": "^1.0.3",
+    "@types/jsonwebtoken": "^9.0.6",
+    "@types/jwt-decode": "^3.1.0",
+    "axios": "^1.7.3",
```

Comment:
Are you using axios? Props for using inbuild fetch function. But when not using axios anymore it should not be in the package json.